pkgs/development/python-modules/geopy/default.nix

```diff
@@ -1,31 +1,33 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
-, isPy27
+, fetchFromGitHub
+, isPy3k
 , mock
 , tox
 , pylint
+, geographiclib
 }:
 
 buildPythonPackage rec {
-  pname = "geopy";
-  version = "1.20.0";
-  disabled = !isPy27;
+    pname = "geopy";
+    version = "2.0-unstable-2019-11-10";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
-  };
+    disabled = !isPy3k; # only Python 3
+    doCheck = false; # too many checks
 
-  doCheck = false;  # too much
+    buildInputs = [ mock tox pylint ];
+    propagatedBuildInputs = [ geographiclib ];
 
-  buildInputs = [ mock tox pylint ];
+    src = fetchFromGitHub {
+        owner  = pname;
+        repo   = pname;
+        rev    = "531b7de6126838a3e69370227aa7f2086ba52b89";
+        sha256 = "07l1pblzg3hb3dbvd9rq8x78ly5dv0zxbc5hwskqil0bhv5v1p39";
+    };
 
-  meta = with stdenv.lib; {
-    homepage = "https://github.com/geopy/geopy";
-    description = "Python Geocoding Toolbox";
-    license = licenses.mit;
-    broken = true;
-  };
-
-}
+    meta = with stdenv.lib; {
+        homepage = "https://github.com/geopy/geopy";
+        description = "Python Geocoding Toolbox";
+        license = licenses.mit;
```

Comment:
Would you like to be the maintainer for this package?

Reply:
If this is of any help, sure

Reply:
Yes, it would be great if somebody updated/changed this package that you could be a second reviewer, who is familiar with this package.